### PR TITLE
Long arguments

### PR DESCRIPTION
**Pull request by Earthcomputer**
Adds a long argument type to be along with the other default primitive argument types (integer, double, etc.). I have needed this a couple of times already in a couple of Minecraft mods and thought this ought to go in brigadier itself rather than create it myself for every mod I make.

### CODE REVIEW

File: src/main/java/com/mojang/brigadier/arguments/LongArgumentType.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+import com.mojang.brigadier.StringReader;
+import com.mojang.brigadier.context.CommandContext;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+public class LongArgumentType implements ArgumentType<Long> {
+    private static final Collection<String> EXAMPLES = Arrays.asList("0", "123", "-123");
```
Comment (Pokechu22) [inline src/main/java/com/mojang/brigadier/arguments/LongArgumentType.java]: You don't declare `getExamples`, so this ends up being unused.

Comment (Earthcomputer) [reply src/main/java/com/mojang/brigadier/arguments/LongArgumentType.java]: Same in `IntegerArgumentType` also, I just copied from that and didn't notice. Both need to be fixed.

Comment (boq) [reply src/main/java/com/mojang/brigadier/arguments/LongArgumentType.java]: `IntegerArgumentType` was already fixed, though after you commit was merged: https://github.com/Mojang/brigadier/blob/master/src/main/java/com/mojang/brigadier/arguments/IntegerArgumentType.java#L90

Comment (Earthcomputer) [reply src/main/java/com/mojang/brigadier/arguments/LongArgumentType.java]: See #16

**Pull request merged**